lib/xing/snapshot.rb

```diff
@@ -0,0 +1,4 @@
+require 'xing/snapshot/local_site_snapshot'
```

Comment:
I agree that snapshot should be consolidated, but should it be under services/snapshot? I'm not sure, just asking.